src/elm/Page/Community.elm

```diff
@@ -145,32 +143,27 @@ view loggedIn model =
                     Page.fullPageGraphQLError (t "community.objectives.title") e
 
                 Loaded community editStatus ->
```

Comment:
We don't need `editStatus` anymore, but I already removed it in `feature/claimer-flow-with-photo` so we can leave it here for now. It will be removed alongside with `EditStatus` type while merging `new_claims`.